graviti/client/commit.py

```diff
@@ -29,13 +32,31 @@ def commit_draft(  # pylint: disable=unused-argument
         description: The draft description.
         tag: The tag name.
 
-    Return:
+    Examples:
+        >>> response = commit_draft("<URL>", "<YOUR_ACCESSKEY>", "<DATASET_ID>", 2, "<TITLE>")
+        >>> response
+        {
+            "commitId": "<COMMIT_ID>"
+        }
+
+    Returns:
         The response of OpenAPI.
 
     """
+    url = urljoin(url, f"gatewayv2/tensorbay-open-api/v1/datasets/{dataset_id}/commits")
```

Comment:
`gatewayv2/tensorbay-open-api/v1/` is a fixed prefix, should set it as a constant?